src/discordcr/voice.cr

```diff
@@ -90,6 +90,19 @@ module Discord
       @udp.send_audio(buf, @sequence, @time)
     end
 
+    # Utility function that runs the given block and measures the time it takes,
+    # then sleeps the given time (or 20 ms by default) minus that time. This is
+    # useful because (in most cases) voice data should be sent to Discord at a
+    # rate of one frame every 20 ms, and if the processing and sending takes
+    # a certain amount of time, then noticeable choppiness can be heard.
+    def self.timed_run(total_time = 20.milliseconds, &block)
```

Comment:
See above again; we still need `timed_run` in case the user wants to only send one audio packet. It may not be useful in the general situation but the spirit of the lib is to give the user a lot of control, so I'd keep this method regardless.